README.md

```diff
@@ -45,3 +45,10 @@ If you make changes to the models.py or other schemas, you need to run a migrati
 1. Set the FLASK_APP environment variable `export FLASK_APP=run.py` or `ENV:FLASK_APP = "run.py"`
 2. Run the migration `flask db migrate`
 3. Upgrade to the latest migration `flask db upgrade`
+
+Before committing, please lint your code:
+
+```sh
+pip install --upgrade flake8
```

Comment:
If flake8 is installed via virtualenv, is upgrading it necessary? Why do we need to run this command before running `flake8 app`?

Reply:
And to clarify, I'm legitimately asking because I don't know, not because I think we should remove this line. I'm sure it's fine, I just want to know why it's necessary.

Reply:
It's not necessary to run it right before `flake8`, as long as it's been installed somehow first.

Some people might not install via virtualenv (I haven't for this PR). And it might be fine without the upgrade, but I generally include it in instructions to ensure the tools are kept up-to-date.

We could remove the upgrade if you prefer.

Reply:
No, I like it as is. I just assumed that anyone who gets to the bottom of the README would have followed the instructions and used virtualenv. But if they're knowledgable enough to know what kind of difference it makes, then they can adjust their command accordingly.

This seems like a good default for those who are kind of blindly following instructions and cargo culting trying to figure this stuff out.